Makefile

```diff
@@ -56,6 +56,11 @@ targets:
 # Build targets
 # -------------
 
+.PHONY: prepare
+prepare: ## Install dependencies
```

Comment:
@simonprev I added a `prepare` target in the "Build" scope, but I'm not sure of the naming convention…